docker/base/os/ubuntu/Dockerfile.focal

```diff
@@ -1,8 +1,8 @@
-# This is the dockerfile for generating base images for ubuntu Buster
+# This is the dockerfile for generating base images for ubuntu Focal Fossa
 # contains the minimal dependencies needed for V to run
 FROM ubuntu:20.04
```

Comment:
If you want to keep up with the latest, just remove the version number.  Whenever the image is built, it will use `ubuntu:latest`.

Reply:
ok but in this case it could be useful to someone to have an already made image for Ubuntu 20.04 (still supported and used by many).